hoot-core-test/src/test/cpp/hoot/core/io/OsmXmlReaderTest.cpp

```diff
@@ -134,14 +128,11 @@ class OsmXmlReaderTest : public CppUnit::TestFixture
         HOOT_STR_EQUALS("Unknown2", map->getWay(-51)->getStatus().toString());
         HOOT_STR_EQUALS("Input003", map->getWay(-14)->getStatus().toString());
         HOOT_STR_EQUALS("Input004", map->getWay(-15)->getStatus().toString());
-
-        OsmXmlWriter writer;
-        writer.write(map, "output.osm");
     }
 
     void runUncompressTest()
     {
-      const std::string cmd("gzip -c test-files/ToyTestA.osm > test-files/ToyTestA_compressed.osm.gz");
+      const std::string cmd("gzip -c test-files/ToyTestA.osm > test-output/ToyTestA_compressed.osm.gz");
```

Comment:
Writing test output to the `test-files` directory instead of `test-output` isn't a good thing.